nautobot-app-chatops/hooks/post_gen_project.py

```diff
@@ -68,6 +68,9 @@
         for folder in folders_to_remove:
             (_ADDONS_PATH / folder).rmdir()
 
+        (_PROJECT_PATH / "docs/models/{{ cookiecutter.model_class_name.lower() }}.md").unlink()
+        (_PROJECT_PATH / "docs/models").rmdir()
```

Comment:
Why can't we add these to `files_to_remove` and `folders_to_remove`?

Reply:
Because `files_to_remove` and `folders_to_remove` uses `_ADDONS_PATH` which in the case of dev-example would look like `nautobot-app-dev-example/nautobot_dev_example/*` which would render for docs to be `nautobot-app-dev-example/nautobot_dev_example/docs/models`, but what we want is `nautobot-app-dev-example/docs/models`. Alternatively, I could rewrite to append _ADDONS_PATH to all of the other files/folders to remove.  This just seemed to be the easiest way to accomplish it.